### PR TITLE
Exponents

### PR DESCRIPTION
**Pull request by nmsderp**
Adds exponents to fusion engine,

### CODE REVIEW

**Comment by zenthm** [comment]
What the heck? There's a standard `math` library in Python, no need to add one for fusion...

**Comment by dimkauzh** [comment]
Yeah zenthm is actually right, this is not really needed. Sorry

**Comment by onepersonintheuniverse** [comment]
TL;DR:  All of this *is in* the standard math lib.

You don't even *need* the Python standard math library for exponents, just use the `**` operator (which also supports floating-point and can do nth-roots for n ∈ ℝ (ℝ being the real numbers, or every single decimal number, such as sqrt(2) and pi. And, the floor function already exists in the standard math lib as `math.floor(x)` which - I am assuming you guessed it - takes the floor of `x`. Plus, there is `math.ceil(x)` as well, with `abs(x)` (the absolute value, also denoted as |x|) is *built-in*, so do not think of adding these, please. And the way to access the standard math lib is `import math`, and despite adding a couple of bytes, it still gives you *way more* than this FE std math lib.

# Sorry for the text wall.
<details>
<summary>#42</summary>
Codename Everhanging Math
</details>
